src/com/facebook/buck/ide/intellij/DefaultIjLibraryFactory.java

```diff
@@ -154,6 +161,12 @@ public void apply(
           .ifPresent(
               sourcePath -> library.addSourceJars(libraryFactoryResolver.getPath(sourcePath)));
       arg.getJavadocUrl().ifPresent(library::addJavadocUrls);
+
+      Path aarUnpackPath = BuildTargets.getScratchPath(
+          projectFilesystem, targetNode.getBuildTarget(), "__unpack_%s#aar_unzip__");
+      if (aarUnpackPath != null) {
+        library.addClassPaths(Paths.get(aarUnpackPath.toString(), "res"));
```

Comment:
As I understand AAR must contain "res". Can you add a comment about that to help with understanding where this constant is coming from